katalog/node-exporter/deploy.yml

```diff
@@ -69,8 +98,8 @@ metadata:
 spec:
   clusterIP: None
   ports:
-  - name: http
+  - name: https
     port: 9100
-    targetPort: 9100
+    targetPort: https
```

Comment:
Same as above, https? is it enforcing https? is it with a self signed certificate?

Reply:
`kube-rbac-proxy` creates a self-signed certificate just to provide encryption.

Reply:
<img width="1413" alt="Captura de pantalla 2020-03-04 a las 9 01 50" src="https://user-images.githubusercontent.com/14357604/75857655-d7146f00-5df6-11ea-86de-5545e5f9ffc6.png">
It is working!